src/components/styles/Layout.tsx

```diff
@@ -33,6 +34,20 @@ export const PageTitle = styled(H2)`
   margin-bottom: 16px;
 `;
 
+export const TableContainer = styled("div")`
+  display ${(props: { hide: boolean }): string => (props.hide ? "none" : "")}
```

Comment:
I think you might be missing a `:` also it might be better to do something like this.

`${(props: {hide: boolean}): string => (props.hide && 'display:none;')}`

This way if hide is false the css wont resolve to `display: `